src/util/suri.js

```diff
@@ -36,6 +36,10 @@ export function parseSURI (suri) {
  */
 
 export function constructSURI ({ derivePath = '', password = '', phrase = '' }) {
+
+  if(derivePath === null || phrase === null) {
```

Comment:
I think we should allow to have the `derivePath` to be null. Either way the fact that it's initialized to an empty string, It actually can't be null at this stage can it?

Reply:
undefined would mean the value wasn't set, but null is an explicit assignment, so it wouldn't use the default empty string values. But yeah derivePath could be null, I'll fix that